vita3k/codec/include/codec/state.h

```diff
@@ -68,6 +76,7 @@ struct DecoderState {
     virtual bool send(const uint8_t *data, uint32_t size) = 0;
     virtual bool receive(uint8_t *data, DecoderSize *size = nullptr) = 0;
     virtual uint32_t get_es_size();
+    virtual DecoderColorSpace get_color_space();
```

Comment:
If this is only used by the Mjpeg decoder, you can instead make it a function of the Mjpeg decoder and modify the MjpegState to have a MjpegDecoderState pointer instead of a DecoderState pointer (not like it can be any of type of decoder anyway).

Reply:
I thought the same function could be used for color space issues in other decoders, but that's just a guess, and I'm sure the implementation details will be different when working with it.
As suggested, I will move this function to the Mjpeg Decoder. If there is a need for a similar function in other decoders, I'll think about it later.